galley/pkg/config/schema/ast/ast_test.go

```diff
@@ -60,8 +58,10 @@ transforms:
 				Collections: []*Collection{
 					{
 						Name:         "istio/meshconfig",
-						Proto:        "istio.mesh.v1alpha1.MeshConfig",
-						ProtoPackage: "istio.io/api/mesh/v1alpha1",
+						VariableName: "IstioMeshconfig",
```

Comment:
Same. Also not quite clear to me what VariableName represents?

Reply:
Variable name is used to create the name of the generated instance in `collections.gen.go`. It can be manually specified in the yaml file .... if not, we use a default based on the collection name.